build.gradle

```diff
@@ -94,7 +94,13 @@ configure(javaProjects) {
             dependency("org.apache.commons:commons-exec:1.3")
             dependency("org.bitbucket.b_c:jose4j:0.5.4")
             dependency("org.dbunit:dbunit:2.5.3")
+            dependencySet(group: "org.jruby", version:"9.1.12.0") {
```

Comment:
This is the dependency management section. They're not being added to the project in any way right now other than the versions are locked if we decide to ever add them. In this case I was testing the `ScriptEngine` implementations they had and decided to leave the dependency lock in just in case.